.github/workflows/actions/deploy/action.yml

```diff
@@ -1,146 +1,78 @@
-            echo "control=dev" >> $GITHUB_OUTPUT
-            echo "healthcheck=${{env.PAAS_APPLICATION_NAME}}-dev" >> $GITHUB_OUTPUT
-            echo "key=schools-experience.dev.terraform" >> $GITHUB_OUTPUT
-            echo "docker_image=${{env.DOCKER_REPOSITORY}}:sha-${{ steps.sha.outputs.short}}" >> $GITHUB_OUTPUT
-        fi
-
-        if [ "${{inputs.environment }}" == "Staging" ]
-        then
-            echo "control=staging" >> $GITHUB_OUTPUT
-            echo "healthcheck=${{env.PAAS_APPLICATION_NAME}}-staging" >> $GITHUB_OUTPUT
-            echo "key=schools-experience.staging.terraform" >> $GITHUB_OUTPUT
-            echo "docker_image=${{env.DOCKER_REPOSITORY}}:sha-${{ steps.sha.outputs.short}}" >> $GITHUB_OUTPUT
-        fi
-
-        if [ "${{inputs.environment }}" == "Production" ]
-        then
-            echo "control=production" >> $GITHUB_OUTPUT
-            echo "healthcheck=${{env.PAAS_APPLICATION_NAME}}-production" >> $GITHUB_OUTPUT
-            echo "key=schools-experience.production.terraform" >> $GITHUB_OUTPUT
-            echo "docker_image=${{env.DOCKER_REPOSITORY}}:sha-${{ steps.sha.outputs.short}}" >> $GITHUB_OUTPUT
-        fi
-
-    - uses: Azure/login@v1
-      with:
-        creds: ${{ inputs.AZURE_CREDENTIALS }}
-
-    - uses: DfE-Digital/keyvault-yaml-secret@v1
-      id: keyvault-infra-secret
-      with:
-        keyvault: ${{ inputs.KEY_VAULT}}
-        secret: SE-INFRA-SECRETS
-        key: LOGIT-API
+        az aks get-credentials -g s189t01-tsc-ts-rg -n s189t01-tsc-test-aks
 
-    - uses: DfE-Digital/keyvault-yaml-secret@v1
-      id: keyvault-yaml-secret
-      with:
-        keyvault: ${{ inputs.KEY_VAULT}}
-        secret: SE-SECRETS
-        key: SECURE_USERNAME , SECURE_PASSWORD
-
-    - uses: hashicorp/setup-terraform@v2
-      with:
-        terraform_version: 1.2.8
-
-    - name: Terraform ( ${{inputs.environment}} )
+    - name: Terraform init, plan & apply
       shell: bash
       run: |
-        cd terraform/paas && pwd
-        terraform init -backend-config=${{steps.variables.outputs.control}}.bk.vars -backend-config="key=${{steps.variables.outputs.key}}.tfstate"
-        terraform apply -var-file=${{steps.variables.outputs.control}}.env.tfvars -auto-approve
+        envval="${{ inputs.environment }}"
+        echo "envval is ${envval}"
+        make ci ${envval} terraform-apply PR_NUMBER=${{ inputs.pr }}
       env:
-        ARM_ACCESS_KEY: ${{ inputs.ARM_ACCESS_KEY  }}
-        TF_VAR_paas_docker_image: ${{ steps.variables.outputs.docker_image}}
-        TF_VAR_AZURE_CREDENTIALS: ${{ inputs.AZURE_CREDENTIALS }}
+        IMAGE_TAG: ${{ steps.sha.outputs.short }}
 
-    - name: Smoke tests
+    - name: Retrieve DSI hostname
+      id: dsi_hostname
       shell: bash
       run: |
-        if [ "${{steps.variables.outputs.RUN_TEST}}" == "true" ]
-        then
-               tests/confidence/healthcheck.sh  "${{steps.variables.outputs.healthcheck}}"  "${{ steps.sha.outputs.short }}"
```

Comment:
This smoke test is useful. Can we make it work or create a new card to add it back?

Reply:
A new card created for smoke test